src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__cell calendar__cell--start-day_mon"></div>
```

Comment:
start-day should be a modifier for the calendar

```suggestion
    <div class="calendar calendar--month-length_31 calendar--start-day_mon">
      <div class="calendar__cell"></div>
```